src/prognostic_equations/gm_sgs_closures.jl

```diff
@@ -19,7 +19,7 @@ function smagorinsky_lilly_length(c_smag, N_eff, dz, Pr, ϵ_st)
     return N_eff > FT(0) ?
            c_smag *
            dz *
-           max(0, 1 - N_eff^2 / Pr / 2 / max(ϵ_st, eps(FT)))^(1 / 4) :
+           max(FT(0), 1 - N_eff^2 / Pr / 2 / max(ϵ_st, eps(FT)))^(1 / FT(4)) :
```

Comment:
`max` does type promotions, so this is actually not needed.

Reply:
Yes, the `FT(0)` is not needed here, and I've removed it. However, the `FT(4)` is needed, since `1 / 4` is always a `Float64`.